source/Canvas.js

```diff
@@ -32,13 +33,19 @@ class Canvas {
     }
   }
 
-  importTiles(tiles) {
+  importTiles(tiles, cartogramArea) {
+    this._mapGraphic.resetBounds()
     this._gridGraphic.importTiles(tiles)
+    this._cartogramReady = cartogramArea
+    exporter.cartogramArea = this._cartogramArea
     this._cartogramReady = true
   }
 
-  updateTiles() {
-    this._gridGraphic.populateTiles(this._mapGraphic)
+  updateTiles(properties) {
+    if (typeof properties !== 'undefined') {
```

Comment:
it can get called from two other places that may or may not supply properties. so we save the initial properties requested via `computeCatogram` and reuse them